### DISCUSSION TITLE
Planes created on GPS coordinates

### DISCUSSION THREAD

**Discussion by mpapado3** [CariusLars/ar_flutter_plugin#77] [Q&A] [answered]
Hello,

Can we use this Flutter plugin to create planes on certain coordinates and present them on device screen only when the user point his device to certain direction and respond to user clicks?

I am thinking of using it creating an AR city guide and to present POI on the device.

Thank you

**Comment by CariusLars** [comment, marked as answer]
Hey @mpapado3, this is not currently possible, but a long time ago a few first steps were taken into that direction [see this stale branch](https://github.com/CariusLars/ar_flutter_plugin/commit/2368d38c1d1fe04133b90a5afc6ed5a17d985b96). 

Feel free to use this as a starting point if you want to add this functionality to the plugin :)
